### PR TITLE
Removed unwanted spaces in group name

### PR DESCRIPTION
**Pull request by danimegar**
| Related issue |
| --- |
| #659 |

## Description
I removed the unwanted spaces for the reason explained in the issue.

### CODE REVIEW

File: rules/0580-win-security_rules.xml [hunk 1]
```diff
@@ -1133,7 +1133,7 @@
 
  <!-- MS IPSec rules -->
 
-<group name="windows, windows_security, ipsec,">
+<group name="windows,windows_security, ipsec,">
```
Comment (vikman90) [inline rules/0580-win-security_rules.xml:1136]: Another leftover whitespace before `ipsec`.

Comment (danimegar) [reply rules/0580-win-security_rules.xml]: Fixed.

File: rules/0590-win-system_rules.xml [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 <var name="MS_FREQ">8</var>
 
-<group name="windows, windows_system">
+<group name="windows,windows_system">
```
Comment (vikman90) [inline rules/0590-win-system_rules.xml:11]: Missing trailing comma `,`.

Comment (danimegar) [reply rules/0590-win-system_rules.xml]: Fixed

**Pull request merged**